content/en/docs/reference/kubernetes-api/common-parameters/common-parameters.md

```diff
@@ -58,6 +58,32 @@ When present, indicates that modifications should not be persisted. An invalid o
 
 
 
+## fieldValidation {#fieldValidation}
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
```

Comment:
Should we mention that we're only ever returning strict errors when there is no non-strict errors?